test/master-playlist-controller.test.js

```diff
@@ -1973,6 +1974,7 @@ function(assert) {
   Playlist.seekable = () => {
     return videojs.createTimeRanges(mainTimeRanges);
   };
+  this.masterPlaylistController.masterPlaylistLoader_.master = media;
```

Comment:
Not a big deal, but may be worth adding a comment saying it's just a placeholder, or putting in a different value, since the master should always reference either a master playlist object or a skeleton of one created to wrap around a media playlist.